docs/application/tizen-studio/common-tools/command-line-interface.md

```diff
@@ -204,7 +204,7 @@ tizen clean [-- <project directory>]
 
 - Clean the project.
 
-  Windows®:
+  Windows&reg;:
 
   ```
   > tizen clean -- C:\Users\workspace\basic
```

Comment:
The **Options** table below this line is broken.

Reply:
Done